packages/component-library/stories/button.notes.md

```diff
@@ -0,0 +1,22 @@
+# Button Component
+
+## Standard
+
+The Standard button uses the standard styles for the CIVIC platform. Most CIVIC projects use the standard button, and edit the label.
+
+## Custom
+
+The Custom button allows the developer to edit these styles of the Standard button, and the label:
+
+- Color theme
+- Transition
+- Text label
+
+## Custom Non-text Label
+
+Use the Non-text Label (e.g., emojis or symbols) button if the label isn't readable by accesibile technologies, e.g. it is an emoji or other image. Use the aria-label attribute to give the button an accessible name. See https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html for more information about the aria-label.
+
+This story allows the developer to edit these attributes of the Standard button:
+
+- Non-text label
+- aria-label
```

Comment:
I took a stab at editing this a little bit more -- let me know what you think

# Button Component

## Standard

The Standard story shows uses the standard usage of the Button for the CIVIC platform. Button includes standard styling.

## Custom

The Custom story shows all possible properties that can be passed to Button component for customization.

- Color theme
- Transition
- Text label

## Example: Non-text Label

This Non-text Label example shows how to use Button if the label isn't readable by accessible technologies, e.g. it is an emoji or other image. Use the aria-label attribute to give the button an accessible name. [More information about aria-label](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html).

This story allows the developer to edit these attributes of the Standard button:

- Non-text label
- aria-label